### PR TITLE
Fixing Night Mode [WIP]

### PR DESCRIPTION
**Pull request by bahaa96**
### Fixing Night Mode [WIP] #1013 
Where do you prefer to place the site language toggle button in the home page?
capture

### Checklist

- [x] Unit tests written
- [x] Manually tested
- [x] Prettier & ESLint were run
- [ ] New dependencies are included in `package-lock.json`

### Screenshot
capture

### CODE REVIEW

File: src/styles/nightmode.scss [hunk 1]
```diff
@@ -1,7 +1,20 @@
...
+  a.end   {
+    color: #fff;
+  }
+  nav {
+    background: #32312C;
+    ul li a {
+      color: #fff;
+    }
+  }
+  .text-translation small {
```
[the remark this answers is not in the document]
Comment (bahaa96) [reply src/styles/nightmode.scss]: OK I'll do all of this. Have you noticed the question in the first line of the PR description, you were asking to have a button to toggle the night mode in the home page. where do you prefer to place that button ?

[the remark this answers is not in the document]
Comment (mmahalwy) [reply src/styles/nightmode.scss]: Hmm, oops. I may have misunderstood something. No need for that button!

**Comment by naveed-ahmad** [comment]
rebuild

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32902

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32903

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32904

**Comment by naveed-ahmad** [comment]
There is some progress in this PR mashAllah, lets merge this and improve the night mode with next PR. Going to merge this.

**Comment by c16u** [comment]
When will this be applied to the site?

**Pull request merged**